lib/screen/name_input_screen.dart

```diff
@@ -29,44 +56,101 @@ class _UserNameScreenState extends State<UserNameScreen> {
                   ),
-                ),
-              ),
-              Spacer(),
-              ElevatedButton(
-                onPressed: _submitName,
-                style: ElevatedButton.styleFrom(
-                  backgroundColor: Colors.greenAccent,
-                  minimumSize: Size(double.infinity, 60),
-                  textStyle: TextStyle(fontSize: 20),
-                ),
-                child: Text('Submit'),
+                  Center(
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      mainAxisSize: MainAxisSize.min,
+                      children: [
+                        Padding(
+                          padding: const EdgeInsets.only(bottom: 20),
+                          child: Text(
+                            'Enter your name:',
+                            style: TextStyle(
+                              fontSize: 18,
+                              fontWeight: FontWeight.bold,
+                              color: Colors.black,
+                            ),
+                          ),
+                        ),
+                        Focus(
+                          onFocusChange: (hasFocus) {
+                            setState(() {
+                            });
+                          },
+                          child: TextField(
+                            controller: _nameController,
+                            decoration: InputDecoration(
+                              enabledBorder: OutlineInputBorder(
+                                borderSide: BorderSide(
+                                  color: Colors.grey, // Default border color
+                                  width: 2.0,
+                                ),
+                                borderRadius: BorderRadius.circular(12.0),
+                              ),
+                              focusedBorder: OutlineInputBorder(
+                                borderSide: BorderSide(
+                                  color: Color(0xFF098EAB), // Border color when focused
+                                  width: 2.0,
+                                ),
+                                borderRadius: BorderRadius.circular(12.0),
+                              ),
+                            ),
+                          ),
+                        ),
+                      ],
+                    ),
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.all(16.0),
+                    child: ScaleTransition(
+                      scale: _isButtonPressed ? _buttonScaleAnimation : AlwaysStoppedAnimation(1),
+                      child: ElevatedButton(
+                        onPressed: _submitName,
+                        style: ElevatedButton.styleFrom(
+                          backgroundColor: Color(0xFF098EAB),
+                          minimumSize: Size(double.infinity, 60),
```

Comment:
Why are you using minimum size here?

Reply:
Using `minimumSize` ensures that the `ElevatedButton` expands to a specified height and width regardless of its content, providing consistent sizing across different screen sizes and layouts. This approach guarantees a uniform appearance for the button in the UI.